Content.Server/Administration/Commands/PlayGlobalSound.cs

```diff
@@ -55,6 +72,6 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 break;
         }
 
-        SoundSystem.Play(filter, args[0], AudioParams.Default);
+        SoundSystem.Play(filter, args[0], volumeOffset == 0 ? audio.WithVolume(-8) : audio);
```

Comment:
Could make the default audio have volume -8 and just pass audio here without checking `volumeOffset`